common/avb_pubkey.c

```diff
@@ -0,0 +1,71 @@
+const char avb_pubkey[] =
```

Comment:
Whoa!

Should it be auto generated based on keys used? It is quite strange to include some public key, in the "common" section without providing instructions how to obtain private part of it. 

In my understanding this should be something deeply product-specific, but I can be wrong there.

Reply:
Actually, I can see that "common/Makefile" issues `bin2c` tool to generate this file. Why it is included in this commit?

Reply:
Hi @lorc, the file was added accidentally. I've removed it.

Done.